### ISSUE TITLE
Huntsville Hospital Health System Medicare Dashboard

### ISSUE THREAD

**Issue by stratabuild** [stratasan/wrike-test#106]
**The Problem we are trying to solve** 
The SAS team creates a Medicare "dashboard" for Huntsville Health System that typically takes about a full week of work for an analyst and for QA. In addition to the amount of time it takes it is susceptible to errors because of the manual process. We have the ability to do this quicker and more efficiently using our data team to automate this report 

**More about the problem**

**Risks** 
Report is built incorrectly/inaccurately 

**Possible Solutions**
Data team creates/automates this dashboard so it can reproduced quickly and accurately 

**User Story** 
I want the Huntsville Medicare Dashboard report to be automated so it can be produced more faster and more accurately

**Assumptions**
SAS team accurately communicates what is needed to build this report

**User Steps** 
_I think CS/SAS will own this step. They may come up with their own method that works best for them._ 
1. User (CS/SAS) requests a new report from data team
2. Report is created by data team
3. SAS reviews report for accuracy. If deemed good then it will be delivered to Huntsville. If not, then back to step one

**Acceptance Criteria**
Recreation of this report: P:\Huntsville Hospital\Projects\Huntsville Dashboard\Huntsville Hospital Medicare Dashboard 2016-2018.xlsx 

**Additional Notes**
The initial building of this report will require collaboration between a member of the SAS team and the data team in order to help the data team member understand what is needed in the report and the parameters around how to create that. Ex. What counties are included for each facility's service area. 

**Link to Sample Report with comments:**  P:\Huntsville Hospital\Projects\Huntsville Dashboard\Huntsville Hospital Medicare Dashboard 2016-2018 with Comments.xlsx

**Link to BB Query Pull:** https://app.stratasan.com/blackbird/result/c9a52740-d8c0-43fa-a4b3-efc977b09821/

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=461614478)
┆Attachments: <a href="https://share.unito.io/6RBh2HEjDdKc8eSCcZc8UcuPV7eF1ruyNgkx32th9ONt">Huntsville Hospital Medicare Dashboard 2016-2018 with Comments.xlsx</a>

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

The next delivery due date is 3/12 for the manual process. The next due date would be 12-14 weeks out.

**Comment by stratabuild** [comment]
➤ Daniel Dreaden commented:

Please leave your contact information
Name
Daniel Dreaden

Email
ddreaden@stratasan.com

Type of request
Other Request

Who is this request for?
Huntsville Hospital and SAS Team

What is the request?
We are asking for an automated repeatable process to be set up to produce the Huntsville Health System Medicare excel dashboard. This excel dashboard uses IPSAF data to provide Huntsville Health System a snapshot for all their facilities and services areas with insights into market share, product lines, top facilities and more. 

Why is the request needed?
This project has averaged around 24 hours of total work the last few times that the SAS has undertaken it. In addition, being done manually opens up a project of this size to errors and inaccuracy. After the initial lift to get this automated is complete this will allow us to get the data to the client quicker and produced with greater accuracy and increase client satisfaction.

When is this request needed by?
Not urgent > 6 Weeks

Request For
Other
Data (APCD, State, Other)

Add screenshots or other files to help

Links to queries/reports
P:\Huntsville Hospital\Projects\Huntsville Dashboard\Huntsville Hospital Medicare Dashboard 2016-2018.xlsx

**Comment by stratabuild** [comment]
➤ Daniel Dreaden commented:

Christian Lawrence Another one to review.  I originally put this one in.  It will take some grooming between myself or a SAS team member with whoever ends up building this on the data team so they can understand what is needed to build the report.

**Comment by stratabuild** [comment]
➤ Daniel Dreaden commented:

Christian Lawrence I believe this one is ready to go as well.  Should we move over?

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Looking at the report, where is all of that data coming from on the P drive that you provided, what are the data sources being used to retrieve that data?

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

I see this in the report: Medical DRG discharges
Stratasan
IPSAF CY 2018
Medicare Inpatient Only
If you can reproduce that query, we may be able to do this in Tableau for them.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Morgan Atkins here is the ticket that I need your help with brainstorming an approach.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Spoke to Hank and budget the time to do it in excel and then do it in tableau as a secondary option while we transition them away. 
Christian Lawrence to get with Brian D to see if it can be done.
We should also plan on doing this with the DST team.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Daniel Dreaden talk to Beth to get the sevice areas that are needed.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Daniel Dreaden provide list of all the items that you need to do in order to pull the report and other types of manual things that are done. provide pain points for Morgan to then see what data services team can do.

**Comment by stratabuild** [comment]
➤ Daniel Dreaden commented:

Christian Lawrence Spoke with Morgan yesterday about this.  I have added comments to the Excel file about "how" to calculate each number and I have also set up a Blackbird Query to pull the data when it is needed.  I have added links to both in the description above.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Daniel Dreaden should this be moved to data science team and assigned to Morgan?

**Comment by stratabuild** [comment]
➤ Daniel Dreaden commented:

Moved to Data Engineering based on our discussion yesterday.

**Comment by stratabuild** [comment]
➤ Christian Lawrence commented:

Daniel and phil have been working on this.
Daniel Dreaden follow up with Hank and determine when a delivery date of this should be.

**Comment by stratabuild** [comment]
➤ Christian commented:

@var-ela Since we are going to park this work, where will your historic notes/ code be stored for reference when we renengage?

**Comment by stratabuild** [comment]
➤ var-ela commented:

@clawdyne I went over what I currently have with Alex yesterday. We agreed on me completing the automation for the selected excel sheets in the tickets and documenting any inconsistencies in some rds and ipsaf data (mostly formatting, but also user related inputs in regards to service areas) which I needed to work around / resolve before getting to the sheets. Will be working on that this sprint session through 6/10, with documentation to show for when it is picked back up come time to run the automated part of the dashboard.
